### ISSUE TITLE
How to use the the zenodo script

### ISSUE THREAD

**Issue by jdemel** [gnuradio/gnuradio-metadata#1]
@willcode I realized I have push access to this repo. Thus, I added an initial working version here.

This issue would be a good place to continue our discussion from gnuradio/gnuradio#5786 .

Most importantly, you brought up the idea to have separate files for each release line.

```
A zenodo dir would be a good place for the tools and user map. There could be one zenodo.json for each branch, e.g., zenodo-maint-3.10.json, or separate directories per branch, or ... ??? TBD: procedures to perform at release time.
```
I'd argue it'd suffice to add this to the latest release for now.

**Comment by willcode** [comment]
Sure, we can try it out for the next 3.10 release in September. It should be close enough to `main` that the contributors are the same. I guess there's no reason to do this with `main` since it doesn't get formally released.

**Comment by jdemel** [comment]
I agree. The tool is not configured to do 2way comparisons. In that case I'd go for a different approach. Anyway, I'd suggest to start with 3.10 releases and later and maybe we can extend this tool to be more sophisticated in the future.

**Comment by dkozel** [comment]
The September release has been done!

**Comment by willcode** [comment]
I did a run and copied the json file in the 3.10 branch before release. It might not be right but it's a start.
